### PR TITLE
Archival notice

### PR DESCRIPTION
**Pull request by justinwilaby**
Archival notice added to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+**What this means:**
+- Pull requests and issue tracking have been disabled.
+- The project will not receive updates and no new features or security updates will be released.
+- The repository is now read-only.
+- The plugin will still be available to use but may be deprecated.
+
+**Why was it archived?**
+This project is considered complete and active development has concluded.
+
```
Comment (sbosio) [inline README.md:15]: I will suggest to change this on this particular repo.

Something like:

This project has been deprecated and replaced by [Heroku JVM Application Deployer](https://github.com/heroku/heroku-jvm-application-deployer). Active development has concluded.

**Review by sbosio — CHANGES_REQUESTED** [review]
Just a small change in this particular repository, because this plugin has a new tool that replaces it, so it will be a good idea to link to that repo.

File: README.md (hunk 1 above)
Comment (justinwilaby) [reply README.md]: Done! thank you.

**Review by sbosio — APPROVED** [review]
Now it's great!

**Comment by justinwilaby** [comment]
[W-14971624](https://gus.lightning.force.com/a07EE00001jVca5YAC)

**Pull request merged**